### PR TITLE
[CET-891] Add system page to README

### PR DESCRIPTION
**Pull request by nikhilunni**
## Change description

> Description here

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,14 +110,24 @@ import { CortexGroupActionItemsWidget } from '@cortexapps/backstage-plugin';
...
-<SidebarItem icon={CortexIcon} to="cortex" text="Cortex" />;
+<Grid item md={4} xs={12}>
+   <CortexGroupActionItemsWidget />
+</Grid>
 ```
 
+10. (Optional) Import `SystemCortexContent` and update [EntityPage.tsx](https://github.com/backstage/backstage/blob/master/packages/app/src/components/catalog/EntityPage.tsx) to add a new catalog tab for Cortex:
+```tsx
+<EntityLayout.Route path="/cortex" title="Cortex">
+<SystemCortexContent />
```
Comment (FoodProduct) [inline README.md:126]: nit:
```suggestion
  <SystemCortexContent />
```

File: README.md [hunk 2]
```diff
@@ -102,14 +110,24 @@ import { CortexGroupActionItemsWidget } from '@cortexapps/backstage-plugin';
...
-8. Add a new sidebar item in [Root.tsx](https://github.com/backstage/backstage/blob/master/packages/app/src/components/Root/Root.tsx)
+9. (Optional) Import `CortexGroupActionItemsWidget` and update [EntityPage.tsx](https://github.com/backstage/backstage/blob/master/packages/app/src/components/catalog/EntityPage.tsx) to add a new group widget for Cortex that shows initiative action items for components owned by that group:
 
 ```tsx
-import { CortexIcon } from '@cortexapps/backstage-plugin';
+import { CortexGroupActionItemsWidget } from '@cortexapps/backstage-plugin';
 
-<SidebarItem icon={CortexIcon} to="cortex" text="Cortex" />;
+<Grid item md={4} xs={12}>
+   <CortexGroupActionItemsWidget />
```
Comment (FoodProduct) [inline README.md:119]: nit: 2 spaces for consistency
```suggestion
 <CortexGroupActionItemsWidget />
```

**Review by FoodProduct — APPROVED** [review]

**Review by cristinabuenahora — COMMENTED** [review]
Will this get auto pulled into here https://www.npmjs.com/package/@cortexapps/backstage-plugin ?

**Review by cristinabuenahora — APPROVED** [review]

**Comment by FoodProduct** [comment]
> Will this get auto pulled into here https://www.npmjs.com/package/@cortexapps/backstage-plugin ?

Are you asking because there is no version bump? If so, I don't know the answer as it depends on the write policy/privileges. It will either replace the existing artifact or get denied. Seems like something to look into (or we can just vbump to be safe).

**Pull request merged**